lib/mix/tasks/phx.gen.context.ex

```diff
@@ -99,7 +99,7 @@ defmodule Mix.Tasks.Phx.Gen.Context do
     {opts, parsed, _} = parse_opts(args)
     [context_name, schema_name, plural | schema_args] = validate_args!(parsed)
 
-    table = Keyword.get(opts, :table, Phoenix.Naming.underscore(context_name) <> "_#{plural}")
+    table = Keyword.get(opts, :table, (Phoenix.Naming.underscore(context_name) |> String.replace("/","_")) <> "_#{plural}")
```

Comment:
I would just change `Phoenix.Naming.underscore(context_name)` to `context_name |> Phoenix.Naming.underscore`. Prefer "pure" values over function calls when starting the pipe chain, [here is the link](https://github.com/rrrene/elixir-style-guide#pipe-chains) the the styleguide definition.